x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/isolate.cy.ts

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  interceptActionRequests,
+  isolateHostWithComment,
+  openAlertDetails,
+  openCaseAlertDetails,
+  releaseHostWithComment,
+  sendActionResponse,
+  waitForReleaseOption,
+} from '../../tasks/isolate';
+import type { ActionDetails } from '../../../../../common/endpoint/types';
+import { closeAllToasts } from '../../tasks/close_all_toasts';
+import type { ReturnTypeFromChainable } from '../../types';
+import { addAlertsToCase } from '../../tasks/add_alerts_to_case';
+import { APP_CASES_PATH } from '../../../../../common/constants';
+import { login } from '../../tasks/login';
+import { indexNewCase } from '../../tasks/index_new_case';
+import { indexEndpointHosts } from '../../tasks/index_endpoint_hosts';
+import { indexEndpointRuleAlerts } from '../../tasks/index_endpoint_rule_alerts';
+
+describe('Isolate command', () => {
+  describe('from Manage', () => {
+    let endpointData: ReturnTypeFromChainable<typeof indexEndpointHosts>;
+    let isolatedEndpointData: ReturnTypeFromChainable<typeof indexEndpointHosts>;
+
+    before(() => {
+      indexEndpointHosts({
+        count: 2,
+        withResponseActions: false,
+        isolation: false,
+      }).then((indexEndpoints) => {
+        endpointData = indexEndpoints;
+      });
+
+      indexEndpointHosts({
+        count: 2,
+        withResponseActions: false,
+        isolation: true,
+      }).then((indexEndpoints) => {
+        isolatedEndpointData = indexEndpoints;
+      });
+    });
+
+    after(() => {
+      if (endpointData) {
+        endpointData.cleanup();
+        // @ts-expect-error ignore setting to undefined
+        endpointData = undefined;
+      }
+
+      if (isolatedEndpointData) {
+        isolatedEndpointData.cleanup();
+        // @ts-expect-error ignore setting to undefined
+        isolatedEndpointData = undefined;
+      }
+    });
+    beforeEach(() => {
+      login();
+    });
+    it('should allow filtering endpoint by Isolated status', () => {
+      cy.visit('/app/security/administration/endpoints');
```

Comment:
can you use the rouing method to get the page's path instead of hard coding it:

```typescript
cy.visit(APP_PATH + getEndpointListPath({ name: 'endpointList' }));
```